deploy/hbase/hbase-site.xml.orig

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration>
+<<<<<<< HEAD
```

Comment:
need to remove residual traces of merge, such as "<<<<<<<< HEAD"

unmodified documents do not need to be submitted